frontend/pages/SoftwarePage/components/SoftwareVulnerabilitiesTable/SoftwareVulnerabilitiesTableConfig.tsx

```diff
@@ -181,8 +147,12 @@ const generateTableConfig = (
           </>
         );
       },
-      Cell: ({ cell: { value } }: ITextCellProps): JSX.Element => (
-        <TextCell value={value ? "Yes" : "No"} />
+      Cell: (cellProps: ICellProps): JSX.Element => (
```

Comment:
I like the `{ cell }` destructuring above – you can do a similar thing here with `{ row }` if you like, not important.

Reply:
I realized there's different patterns for cell prop interfaces, some configs breaking them down into more specific interfaces like ITextCellProps or INumberCellProps. Definitely a good frontend topic to add to the agenda.